src/coreclr/inc/mscoree.idl

```diff
@@ -36,58 +36,18 @@ cpp_quote("EXTERN_GUID(IID_ICLRRuntimeHost4, 0x64F6D366, 0xD7C2, 0x4F1F, 0xB4, 0
 
 typedef HRESULT (__stdcall *FExecuteInAppDomainCallback) (void* cookie);
 
-// By default GC is concurrent and only the base system library is loaded into the domain-neutral area.
 typedef enum {
   STARTUP_CONCURRENT_GC         = 0x1,
 
-  STARTUP_LOADER_OPTIMIZATION_MASK = 0x3<<1,                    // loader optimization mask
-  STARTUP_LOADER_OPTIMIZATION_SINGLE_DOMAIN = 0x1<<1,           // no domain neutral loading
-  STARTUP_LOADER_OPTIMIZATION_MULTI_DOMAIN = 0x2<<1,            // all domain neutral loading
-  STARTUP_LOADER_OPTIMIZATION_MULTI_DOMAIN_HOST = 0x3<<1,       // strong name domain neutral loading
```

Comment:
There is `LoaderOptimizationAttribute` specifying these flags. Now all the flags are deprecated, should the attribute be considered for deprecation?

Reply:
Possibly. I think you can find number of attribute types that were added for compatibility with .NET Framework and that have no effect in .NET Core.